### DISCUSSION TITLE
Login with google

### DISCUSSION THREAD

**Discussion by SakuraIce22** [thatskymod/Sky-CotL-Scripts#61] [General]
Hello, I'm new to this whole thing and just had a little question.
I've just downloaded Canvas and launched it, will login in with my google account erase my exising Sky account?

**Comment by thatskymod** [comment]
@SakuraIce22,

No, logging in to Canvas using your Google credentials will not erase, alter or affect the way your Sky account is linked, unless you explicitly choose to modify these settings. This essentially the same as logging in through the official Sky app, and it's impossible to break something accidentally.

### Important:
Keep in mind, however, that as of January 2024, Google login is not supported by Canvas and will result in an unavoidable server error. This is not an issue we can currently resolve, so you are strongly advised to use an alternative login method instead. Logging in with Nintendo credentials tends to work best.

Let me know if you have further questions or concerns!

Kind regards,
thatskymod

**Reply by SakuraIce22** [reply]
Alright, thank you.

**Comment by thatskymod** [comment]
This question appears to be answered, so it seems appropriate to close this topic.

If you have any more questions, feel free to reach out!
